store/localcachelayer/channel_layer.go

```diff
@@ -70,6 +80,30 @@ func (s LocalCacheChannelStore) InvalidateGuestCount(channelId string) {
 	}
 }
 
+func (s LocalCacheChannelStore) InvalidateAllChannelMembersNotifyPropsForChannel(channelId string) {
+	s.rootStore.doInvalidateCacheCluster(s.rootStore.allChannelMembersNotifyPropsForChannelCache, channelId)
+	if s.rootStore.metrics != nil {
```

Comment:
This metrics counting is already handled by the `doInvalidateCacheCluster` method, so is not needed here.

Reply:
Nevermind, that's not true, my bad.